solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -613,20 +665,42 @@ By default, generated consumer group queue names have the following form:
 ----
 <prefix>/<familiarity-modifier>/<group>/<destination-encoding>/<encoded-destination>
 ----
+The `queueNameExpression` property's default Spring Expression conforms to the above format, however, users can provide any valid Spring Expression in order to generate custom queue names. Valid expressions evaluate against the following context:
+[cols="1m,1", options="header"]
+|===
+| Context Variable
+| Description
+
+| destination
+| The binding’s destination name.
+
+| group
+| The binding’s consumer group name.
+
+| isAnonymous
+| Indicates whether the consumer is an anonymous consumer group
 
-prefix::
+| properties.solace
+| The configured Solace binding properties.
+
+| properties.spring
+| The configured Spring binding properties.
+|===
+WARNING: The expression evaluation context is subject to change and should not be used other than to customize queue names.
```

Comment:
I might have misunderstood a comment from a previous Slack discussion (I had copied the comment in https://sol-jira.atlassian.net/browse/SOL-58750): 
<img width="825" alt="image" src="https://user-images.githubusercontent.com/5495913/155152064-4a55ad6a-439f-470c-9848-c6225cc44c6b.png">

Reply:
The SpEL **expressions** are subject to change. But the SpEL's **evaluation contexts** are not.

If we were to allow changing the evaluation context, then we would break backwards compatibility with users who have provided their own SpEL expressions whenever we changed the contract for the evaluation context.

Reply:
Removing this warning. 
You can suggest any new warnings you feel are necessary on the next review.